pkg/updatestatus/nodeinformer.go

```diff
@@ -303,6 +307,12 @@ func (c *machineConfigPoolSelectorCache) forget(mcpName string) bool {
 	return false
 }
 
+func (c *machineConfigPoolSelectorCache) len() int {
+	c.lock.Lock()
+	defer c.lock.Unlock()
+	return len(c.cache)
+}
```

Comment:
Feels like overkill to have a locked method just to log `...N config pools` - I think a simple "does not belong to any MachineConfigPool" log is just as useful :shrug: